subprojects/docs/src/docs/release/notes.md

```diff
@@ -105,6 +105,17 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.html) for all supported methods.
 
 
+<a name="build-init"></a>
+### Build Init improvements
+
+#### Build Init utilizing version Catalogs
+Version catalogs are the recommended way to centrally define dependency modules, plugins and their versions.
+The [build init plugin](userguide/build_init_plugin.html) now generates projects utilizing version catalogs in the conventional location `gradle/libs.versions.toml`.
```

Comment:
💭 It really feels to me like there should be an opt-out for this.  I imagine many people aren't familiar with Version Catalogs, and when quickly generating new/test projects might not appreciate being forced to use it.

Reply:
The goal if for people to get used to version catalogs.